### ISSUE TITLE
support for external storages

### ISSUE THREAD

**Issue by wmartinmimi** [martinmimigames/little-file-explorer#15]
eg usb, network, etc.

**Comment by StevanWhite** [comment]
Yes. Managing files over USB is pretty minimal functionality for a file manager.

App should
1) allow navigation and basic file operations on USB partition.
2) support multiple partitions
3) handle read-only filesystems and devices appropriately
        (visibly indicate is read-only, warn user that filesystem is read-only, if they try to write to it.)
4) display filesystem labels properly
5) respond gracefully to on-the-fly insertion and removal of device.

Also, if user has selected app to open inserted USB by default, then USB insertion should open the file browser activity for the drive, as directly as possible.

 -----------------------------------
 Some examples of freely-usable (but not all open source) file managers that handle USB pretty well:
 
 * Files (google.com)
 * Cx File Explorer (cxinventor.com)
 * Owlfiles (skype.com)
----------------------------------- 
Android supports FAT32/Ext3/Ext4 file system.
Most of the the latest smartphones and tablets support exFAT file system.
